pkg/app/piped/controller/scheduler.go

```diff
@@ -635,6 +650,21 @@ func (s *scheduler) reportDeploymentCompleted(ctx context.Context, status model.
 	return err
 }
 
+func (s *scheduler) getMentionedAccounts(ctx context.Context, event model.NotificationEventType) ([]string, error) {
+	ds, err := s.targetDSP.GetReadOnly(ctx, ioutil.Discard)
```

Comment:
This causes a crash when `s.targetDSP` was not configured in some cases such as below:
 
https://github.com/pipe-cd/pipe/blob/96548e661633a09377ed5eca1dc6ff82397c128e/pkg/app/piped/controller/scheduler.go#L223-L227

Reply:
eee40f8